### PR TITLE
main.py update

### PR DESCRIPTION
**Pull request by LukeG294**
Just a few extra features including a list of commands and a BrainBot introduction 👍

### CODE REVIEW

File: main.py [hunk 1]
```diff
@@ -77,6 +77,12 @@ async def on_chat(msg):
                 elif msg.text.lower().startswith("!version"):
                     console.log(f"Telling {user.get_username()} the current version")
                     await send_message(f"BrainBot v{__version__}", bot_chat)
+                elif msg.text.lower().startswith("!WhoisBrainBot?"):
```
Comment (bsoyka) [inline main.py:80]:
```suggestion
                elif msg.text.lower().startswith("!intro?"):
```

File: main.py [hunk 2]
```diff
@@ -77,6 +77,12 @@ async def on_chat(msg):
...
+                    console.log(f"Telling {user.get_username()} who I am.")
+                    await send_message(f"Hi! I'm BrainBot. I'm a fun, engagement-increasing bot made by @bsoyka. Ask me for a list of commands if you'd like by saying !commands")
+                elif msg.text.lower().startswith("!commands"):
+                    console.log(f"Telling {user.get_username()} my commands.")
+                    await send_message(f"!topic, !version, !restart (admin only), !topicbypass (admin only), !commands, WhoisBrainbot")
```
Comment (bsoyka) [inline main.py:85]:
```suggestion
                    await send_message(f"!topic, !version, !commands, !intro, !restart (admin only), !topic bypass (admin only)")
```

File: main.py (hunk 1 above)
Comment (LukeG294) [reply main.py:80]: Good idea, would you still need the question mark in that case?

File: main.py (hunk 2 above)
Comment (LukeG294) [reply main.py:85]: Yep, looks much better like that.

File: main.py (hunk 1 above)
Comment (bsoyka) [reply main.py:80]: Fixed!

**Review by bsoyka — APPROVED** [review]

**Comment by bsoyka** [comment]
Merging now, thanks for your contribution!

**Comment by bsoyka** [comment]
@all-contributors please add @bsoyka for code, idea, doc, maintenance, review

**Comment by bsoyka** [comment]
@all-contributors please add @LukeG294 for code, idea

**Pull request merged**